app/views/casework/send_decisions/new.html.erb

```diff
@@ -11,9 +11,11 @@
       <%= @crime_application.applicant.full_name %>
     </h1>
 
-    <h2 class="govuk-heading-m">
-      <%= t '.title', count: @decisions.size %>
-    </h2>
+    <% unless @crime_application.work_stream == 'non_means_tested' %>
```

Comment:
You can use your new concern ☝️ `@crime_application.non_means?`

Reply:
I forgot 😭 thank you